<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bailar Music</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Unchanged1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Medium</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Note Played:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="C">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>61</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Iskoola Pota</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#C
{
    background-color: transparent;
    background-image: url(:/new/prefix1/A_new.png) ;
    background-repeat: none;    
    border: none;
	color: white;

}

QPushButton#C:hover
{
	color: black;
	background-image: url(:/new/prefix1/A_hov.png);

}
</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="D">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>61</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Iskoola Pota</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#D
{
    background-color: transparent;
    background-image: url(:/new/prefix1/b_new.png) ;
    background-repeat: none;    
    border: none;
	color: white;
}


QPushButton#D:hover
{
	color: black;
    background-image: url(:/new/prefix1/b_hov.png) ;  
}</string>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="E">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>61</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Iskoola Pota</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#E
{
    background-color: transparent;
    background-image: url(:/new/prefix1/c_new.png) ;
    background-repeat: none;    
    border: none;
	color: white;
}

QPushButton#E:hover
{
	color: black;
    background-image: url(:/new/prefix1/c_hov.png) ;

}</string>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>61</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Iskoola Pota</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#F
{
    background-color: transparent;
    background-image: url(:/new/prefix1/A_new.png) ;
    background-repeat: none;    
    border: none;
color: white;
}

QPushButton#F:hover
{
	color: black;
	background-image: url(:/new/prefix1/A_hov.png);
}
</string>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="G">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>61</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Iskoola Pota</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#G
{
    background-color: transparent;
    background-image: url(:/new/prefix1/b_new.png) ;
    background-repeat: none;    
    border: none;
	color: white;
}


QPushButton#G:hover
{
	color:black;
    background-image: url(:/new/prefix1/b_hov.png) ;  
}</string>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="A">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>61</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Iskoola Pota</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#A
{
    background-color: transparent;
    background-image: url(:/new/prefix1/b_new.png) ;
    background-repeat: none;    
    border: none;
	color: white;
}


QPushButton#A:hover
{
	color: black;
    background-image: url(:/new/prefix1/b_hov.png) ;  
}</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>61</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Iskoola Pota</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#B
{
    background-color: transparent;
    background-image: url(:/new/prefix1/c_new.png) ;
    background-repeat: none;    
    border: none;
	color: white;
}

QPushButton#B:hover
{
	color: black;
    background-image: url(:/new/prefix1/c_hov.png) ;

}</string>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLabel" name="Unchanged2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Medium</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Most Likely Following Note:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Note1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="Note2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="C_sh">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>21</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#C_sh
{
    background-color: transparent;
    background-image: url(:/new/prefix1/sharp_new.png) ;
    background-repeat: none;    
    border: none;
	color: black;
}

QPushButton#C_sh:hover
{
	color: white;
}</string>
    </property>
    <property name="text">
     <string>C#</string>
    </property>
   </widget>
   <widget class="QPushButton" name="D_sh">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>31</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#D_sh
{
    background-color: transparent;
    background-image: url(:/new/prefix1/sharp_new.png) ;
    background-repeat: none;    
    border: none;
	color: black;
}

QPushButton#D_sh:hover
{
	color: white;
}</string>
    </property>
    <property name="text">
     <string>D#</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F_sh">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>31</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#F_sh
{
    background-color: transparent;
    background-image: url(:/new/prefix1/sharp_new.png) ;
    background-repeat: none;    
    border: none;
	color: black;
}

QPushButton#F_sh:hover
{
	color: white;
}</string>
    </property>
    <property name="text">
     <string>F#</string>
    </property>
   </widget>
   <widget class="QPushButton" name="G_sh">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>31</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#G_sh
{
    background-color: transparent;
    background-image: url(:/new/prefix1/sharp_new.png) ;
    background-repeat: none;    
    border: none;
	color: black;
}

QPushButton#G_sh:hover
{
	color: white;
}</string>
    </property>
    <property name="text">
     <string>G#</string>
    </property>
   </widget>
   <widget class="QPushButton" name="A_sh">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>31</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#A_sh
{
    background-color: transparent;
    background-image: url(:/new/prefix1/sharp_new.png) ;
    background-repeat: none;    
    border: none;
	color: black;
}

QPushButton#A_sh:hover
{
	color: white;
}</string>
    </property>
    <property name="text">
     <string>A#</string>
    </property>
   </widget>
   <widget class="QLabel" name="Unchanged1_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Medium</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Status">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>- - -Idle- - -</string>
    </property>
   </widget>
   <widget class="QLabel" name="Unchanged1_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Medium</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Last Saved:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Save_Date">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>265</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>596</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBailar">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="Save"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuChords">
    <property name="title">
     <string>Chords</string>
    </property>
    <addaction name="Chord_Mode"/>
    <addaction name="actionPossible_chords"/>
   </widget>
   <addaction name="menuBailar"/>
   <addaction name="menuChords"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Chord_Mode">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/jimi.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Chord Mode</string>
   </property>
  </action>
  <action name="Save">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/saveIcon.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionPossible_chords">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/note_picture.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Possible chords</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
